recipes/sos-sas/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "sos-sas" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8611cb450f86f2d07b74ea1043ec0bc40fb1b19a98c8c7aa52db889b12211891
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook >=0.19.4
+    - sas_kernel
+    - saspy
+    - sas7bdat
```

Please, add ``python`` here as well
if it is a ``noarch`` recipe you can do 
```suggestion
    - sas7bdat
    - python >=3.6
```

---

Just did it.